src/python_testing/TC_AVSUM_2_2.py

```diff
@@ -0,0 +1,203 @@
+#       --passcode 20202021
+#       --PICS src/app/tests/suites/certification/ci-pics-values
+#       --trace-to json:${TRACE_TEST_JSON}.json
+#       --trace-to perfetto:${TRACE_TEST_PERFETTO}.perfetto
+#       --endpoint 1
+#     factory-reset: true
+#     quiet: true
+# === END CI TEST ARGUMENTS ===
+
+import random
+
+import chip.clusters as Clusters
+from chip.interaction_model import Status
+from chip.testing.matter_testing import MatterBaseTest, TestStep, async_test_body, default_matter_test_main
+from mobly import asserts
+from TC_AVSUMTestBase import AVSUMTestBase
+
+
+class TC_AVSUM_2_2(MatterBaseTest, AVSUMTestBase):
+    has_feature_mpan = False
+    has_feature_mtilt = False
+    has_feature_mzoom = False
+
+    def desc_TC_AVSUM_2_2(self) -> str:
+        return "[TC-AVSUM-2.2] MptzSetPosition Command Validation"
+
+    def steps_TC_AVSUM_2_2(self) -> list[TestStep]:
+        steps = [
+            TestStep(1, "Commissioning, already done", is_commissioning=True),
+            TestStep(2, "Read and verify MPTZPosition attribute."),
+            TestStep(3, "Validate changes to the Pan value, if supported."),
+            TestStep(4, "Validate changes to the Tilt value, if supported."),
+            TestStep(5, "Validate changes to the Zoom value, if supported"),
+        ]
+        return steps
+
+    def pics_TC_AVSUM_2_2(self) -> list[str]:
+        pics = [
+            "AVSUM.S",
+        ]
+        return pics
+
+    @async_test_body
+    async def test_TC_AVSUM_2_2(self, endpoint=1):
+        cluster = Clusters.Objects.CameraAvSettingsUserLevelManagement
+        attributes = cluster.Attributes
+
+        self.step(1)  # Already done, immediately go to step 2
+
+        feature_map = await self.read_avsum_attribute_expect_success(endpoint=endpoint, attribute="FeatureMap")
+        self.has_feature_mpan = (feature_map & cluster.Bitmaps.Feature.kMechanicalPan) != 0
+        self.has_feature_mtilt = (feature_map & cluster.Bitmaps.Feature.kMechanicalTilt) != 0
+        self.has_feature_mzoom = (feature_map & cluster.Bitmaps.Feature.kMechanicalZoom) != 0
+
+        attribute_list = await self.read_avsum_attribute_expect_success(endpoint=endpoint, attribute="AttributeList")
+
+        if not (self.has_feature_mpan | self.has_feature_mtilt | self.has_feature_mzoom):
+            asserts.fail("One of MPAN, MTILT, or MZOOM is mandatory")
+            self.skip_all_remaining_steps(2)
+
+        self.step(2)
+        asserts.assert_in(attributes.MPTZPosition.attribute_id, attribute_list, "MPTZPosition attribute is mandatory.")
+        mptzposition_dut = await self.read_avsum_attribute_expect_success(endpoint=endpoint, attribute="MPTZPosition")
+        initialPan = mptzposition_dut.pan
+        initialTilt = mptzposition_dut.tilt
+        initialZoom = mptzposition_dut.zoom
+
+        if self.has_feature_mpan:
+            self.step(3)
+            asserts.assert_in(attributes.PanMin.attribute_id, attribute_list,
```

Comment:
Not that I particularly object to more testing, but you already tested the conformance and ranges in 2.1 and we have conformance checks in the IDM section. This is already written, so doesn't need to be removed, but if you have more tests coming, you can omit this if you want and it'll make the tests shorter.